### PR TITLE
Detach global event listeners function

### PR DESCRIPTION
**Pull request by YukiChung**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,11600 +1,8 @@
 {
   "name": "webrecorder",
   "version": "0.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```
Comment (agamm) [inline package-lock.json:4]: This file should not change.

File: src/pages/Content/index.js [hunk 2]
```diff
@@ -13,3 +13,17 @@ function click(selector) {
   const element = document.querySelector(selector) || null;
   if (element) element.click();
 }
+// The function plays a recording when needed
+function playRecording(recordedEvents) {
```
Comment (agamm) [inline src/pages/Content/index.js:17]: This should not appear in this branch and PR. Separate the branches.

**Review by agamm — CHANGES_REQUESTED** [review]
Wait with this PR until the `Stat global event listener` code is merged and talk to Togi before to see that the code will work with his code ;)

File: src/pages/Content/index.js (hunk 2 above)
Comment (itoogii) [reply src/pages/Content/index.js:17]: As Agam advised,  can you please remove this function playRecording(recordedEvents) from this branch as it is in another branch?

File: tests/playground.html [hunk 3]
```diff
@@ -2,12 +2,34 @@
 
 <head>
     <title>Button Counter</title>
+    <script src="http://code.jquery.com/jquery-latest.min.js"></script>
```
Comment (itoogii) [inline tests/playground.html:5]: I think Agam will ask you to create another branch for the playground.html changes. As this branch is for only the detachGlobaEventListeners()

Comment (agamm) [reply tests/playground.html:5]: Yes, nice catch ;)

**Pull request closed without merging**